frontend/packages/kubevirt-plugin/src/models/templates/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './vm-template-yaml-templates';
+export * from './vm-yaml-templates';
```

Comment:
nit: as this is already under `models/templates`, do we need the extra `-templates` at the end of the filename?

Reply:
:+1: